woocommerce/pinterest-for-woocommerce
# Ads credit success modal

### Changes proposed in this Pull Request:



Closes #566 .

Just adding the modal.


### Screenshots:

<img width="808" alt="image" src="https://user-images.githubusercontent.com/17271089/192794484-0518bdb3-f6b7-4058-950e-af6cba931db7.png">


### Detailed test instructions:


1. Use this branch
2. Onboard
3. Observe the modal


### Changelog entry

>
